esp/src/eclwatch/TargetSelectClass.js

```diff
@@ -303,10 +303,6 @@ define([
                 }).then(function (response) {
                     if (lang.exists("TpDropZoneQueryResponse.TpDropZones.TpDropZone", response)) {
                         context.set("options", []);
-                        context.options.push({
-                            label: "&nbsp;",
-                            value: ""
-                        });
```

Comment:
Removing this will prevent the other instances from defaulting to a blank line (if there are any other instances?) - is this your intention?

Reply:
Correct. This is for the IP's in DZ, Despray and Upload. The user is stating that the first IP should be auto filled.